app/scripts/templates/ready.mustache

```diff
@@ -16,19 +24,48 @@
 
     <div class="graphic graphic-checkbox">{{#t}}Success{{/t}}</div>
 
-    {{#serviceName}}
-      <p class="account-ready-service">{{#t}}You are now ready to use %(serviceName)s{{/t}}</p>
-      {{#showProceedButton}}
-      <form novalidate>
-        <div class="button-row">
-          <button id="proceed" type="submit">{{#t}}Proceed{{/t}}</button>
-        </div>
-      </form>
-      {{/showProceedButton}}
-    {{/serviceName}}
-    {{^serviceName}}
-      <p class="account-ready-generic">{{#t}}Your account is ready!{{/t}}</p>
-    {{/serviceName}}
+    {{#isSync}}
+      {{#isAccountUnlock}}
+        <p class="account-ready-service">{{#t}}Firefox Sync will begin momentarily{{/t}}</p>
+      {{/isAccountUnlock}}
+      {{#isForceAuth}}
+        <p class="account-ready-service">{{#t}}Firefox Sync will resume momentarily{{/t}}</p>
+      {{/isForceAuth}}
+      {{#isResetPassword}}
+        <p class="account-ready-service">{{#t}}Firefox Sync will begin momentarily{{/t}}</p>
+      {{/isResetPassword}}
+      {{#isSignIn}}
+        <p class="account-ready-service">{{#t}}Firefox Sync will begin momentarily{{/t}}</p>
+      {{/isSignIn}}
+      {{#isSignUp}}
+        <p class="account-ready-service">{{#t}}You are now ready to use %(serviceName)s{{/t}}</p>
+      {{/isSignUp}}
```

Comment:
same as https://github.com/mozilla/fxa-content-server/pull/3094/files#r40962304 

we should file an issue if you agree